frontend/cypress/integration/common/kiali_logout.ts

```diff
@@ -0,0 +1,19 @@
+import { Given, When, And, Then } from "cypress-cucumber-preprocessor/steps";
```

Comment:
`Given, When` imports are not used in this file, so there is no reason to import them